wca-guidelines.md

```diff
@@ -126,7 +126,6 @@ To be more informative, each Guideline is classified using one of the following
 
 ## <article-9><events><events> Article 9: Events
 
-- 9b+) [ADDITION] The preferred format for the final of an event is "Average of 5" or "Mean of 3", if possible.
```

Comment:
Could we keep "Average of 5" to clarify that combined rounds are not preferred for finals?

Reply:
But are they? I think we use them quite commonly, and I don't see a problem with it.

Reply:
Maybe we can reword it a bit to make that clear?

Reply:
How's this?

```suggestion
- 9b+) [ADDITION] If an event has multiple rounds, every round after the first round (especially the final round) should use the "Average of 5" format.
```

Reply:
I think I would keep the "Mean of 3" too, otherwise LGTM!

Reply:
Ah, sorry, I was somehow thinking 9b+ was "9b1+".

```suggestion
- 9b+) [ADDITION] For an event with multiple rounds, every round after the first round (especially the final round) should be "Average of 5" or "Mean of 3" if one of these is a valid format.
```